comictaggerlib/comiccacher.py

```diff
@@ -174,27 +179,28 @@ def get_search_results(self, search_term: str) -> list[CVVolumeResults]:
             cur.execute("DELETE FROM VolumeSearchCache WHERE timestamp  < ?", [str(a_day_ago)])
 
             # fetch
-            cur.execute("SELECT * FROM VolumeSearchCache WHERE search_term=?", [search_term.lower()])
+            cur.execute("SELECT * FROM VolumeSearchCache WHERE search_term=? AND source_name=?", [search_term.lower(), source_name])
             rows = cur.fetchall()
             # now process the results
             for record in rows:
                 result = CVVolumeResults(
                     {
-                        "id": record[1],
-                        "name": record[2],
-                        "start_year": record[3],
-                        "count_of_issues": record[5],
-                        "description": record[7],
-                        "publisher": {"name": record[4]},
-                        "image": {"super_url": record[6]},
+                        "source": record[0],
+                        "id": record[2],
+                        "name": record[3],
+                        "start_year": record[4],
+                        "count_of_issues": record[6],
+                        "description": record[8],
+                        "publisher": {"name": record[5]},
+                        "image": {"super_url": record[7]},
```

Comment:
I would recommend putting source as the last column it makes this block a one line change